ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -366,9 +371,10 @@ const AssetListControlBar = ({
       Object.keys(allEnabledNetworksForAllNamespaces).length === 1
     ) {
       const chainId = allEnabledNetworksForAllNamespaces[0];
-      return isStrictHexString(chainId)
-        ? (allNetworks[chainId]?.name ?? t('currentNetwork'))
-        : (currentMultichainNetwork.network.nickname ?? t('currentNetwork'));
```

Comment:
This is BAD in BIP-44 world. Since a BIP-44 account can contain EVM + Non-EVM, so this will fallback to EVM is some scenarios 💀